gtsfm/utils/io.py

```diff
@@ -2,7 +2,7 @@
 
 Authors: Ayush Baid, John Lambert
 """
-import json
+import simplejson as json
```

Comment:
what happened to native json?

Reply:
Native JSON in Python outputs np.NaN as "NaN" to JSON files. This is not actually part of the JSON spec, so ReactJS throws an error when reading NaNs. The simplejson library has an ignore_nan flag that outputs "null" for all NaNs (and otherwise behaves nearly identically). 

Source: https://stackoverflow.com/questions/6601812/sending-nan-in-json

Reply:
How does numpy deal with nulls? Can we parse these JSONs? There are some tools that parse them and create plots, and we want to be sure they dont break with nulls.

Reply:
@travisdriver @johnwlambert I remember you running into some issues with NaNs in these metrics.

Reply:
The JSONs can be parsed, but any "null" is turned into a None in Python (see [here](https://stackoverflow.com/questions/49829286/convert-json-null-to-nan-in-simplejson-load)). This unit test I just wrote encapsulates the behavior in simplejson:
![Screenshot from 2022-07-22 21-03-01](https://user-images.githubusercontent.com/30275369/180584411-f9ffe1a3-92cb-4c07-aa29-8c9a4bd69b19.png)

How are NaNs displayed in the existing tools? I assume they are not displayed; if so, I can fix the tools to check that the data they are attempting to display is not None. I looked very deeply into changing the way React parses JSON files and was unable to figure out how to force different parsing behavior, which is why I chose this method. (Also, I can check in that unit test once we resolve this to reduce future confusion.)